openmetadata-ui/src/main/resources/ui/src/components/BotDetails/AuthMechanismForm.tsx

```diff
@@ -640,13 +644,15 @@ const AuthMechanismForm: FC<Props> = ({
           centered
           destroyOnClose
           closable={false}
-          okText="Confirm"
-          title="Are you sure?"
+          confirmLoading={isLoading}
+          okText={t('label.confirm')}
+          title={t('label.are-you-sure')}
           visible={isConfirmationModalOpen}
           onCancel={() => setIsConfirmationModalOpen(false)}
           onOk={handleAccountEmailChange}>
           <Typography.Text>
-            {BOT_ACCOUNT_EMAIL_CHANGE_CONFIRMATION} for {botData.name} bot.
+            {BOT_ACCOUNT_EMAIL_CHANGE_CONFIRMATION} {t('label.for')}{' '}
+            {botData.name} {t('label.bot')}.
```

Comment:
It should be a single message on message.bot-email success with providing botName
Reference: https://www.i18next.com/translation-function/interpolation